tests/Admin/AdminTest.php

```diff
@@ -1589,7 +1589,7 @@ public function testGetFormWithCollectionParentValue(): void
 
     public function testFormAddPostSubmitEventForPreValidation(): void
     {
-        $modelAdmin = new ModelAdmin('sonata.post.admin.model', 'Application\Sonata\FooBundle\Entity\Model', 'SonataFooBundle:ModelAdmin');
+        $modelAdmin = new ModelAdmin('sonata.post.admin.model', '\stdClass', 'SonataFooBundle:ModelAdmin');
```

Comment:
```suggestion
        $modelAdmin = new ModelAdmin('sonata.post.admin.model', \stdClass::class, 'SonataFooBundle:ModelAdmin');
```
Even, if possible we could use the currently preferred syntax for the third argument: 'SonataFooBundle\ModelAdminController'.